addons/project/models/project.py

```diff
@@ -356,9 +356,9 @@ def _read_group_stage_ids(self, stages, domain, order):
              "with Tasks (or optionally Issues if the Issue Tracker module is installed).")
     alias_value = fields.Char(string='Alias email', compute='_compute_alias_value')
     privacy_visibility = fields.Selection([
-            ('followers', 'Invited internal users'),
-            ('employees', 'All internal users'),
-            ('portal', 'Invited portal users and all internal users'),
+            ('followers', 'Invited internal users (private)'),
+            ('employees', 'All internal users (company)'),
```

Comment:
We no longer need this change (for employees item) because the task-3084819, we will remove the ir.rule. 🙂 
```suggestion
            ('employees', 'All internal users'),
```